package.json

```diff
@@ -15,7 +15,8 @@
   },
   "dependencies": {
     "@erc725/erc725.js": "0.14.1",
-    "@lukso/lsp-factory.js": "1.0.2-alpha.6",
```

Comment:
I see you merged in other pending PR about lsp-factory. That might happen when you have PR dependency but let's just wait till the other one is resolved and merged into `main` branch first.

Reply:
Okay, thanks